src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2053,7 +2048,22 @@ export class ResultsViewPageStore {
                 ),
         },
         { samples: {}, patients: {} }
-    );
+    );*/
+    readonly coverageInformation = remoteData<CoverageInformation>({
+        await: () => [
+            this.molecularProfilesInStudies,
+            this.sampleKeyToSample,
+            this.patients,
+            this.genes,
+        ],
+        invoke: () =>
+            getCoverageInformation(
+                this.molecularProfilesInStudies.result!,
```

Comment:
Do we need gene-panel data for all molecular profiles in studies or just the selected profiles?

Reply:
good catch, yeah I think you're right